src/player/features/volume/model/VolumeFeature.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, expect, test } from "vitest";
+
+import { Player } from "@/player/entities/player";
+import { InMemoryAudioManager } from "@/player/lib/audio-manager";
+import { scaleVolumeToUnitRange } from "../utils/ScaleVolume";
+import { VolumeFeature } from "./VolumeFeature";
+
+describe("SeekFeature", () => {
+  test("should change the volume", () => {
+    const currentTime = 10;
+    const currentVolume = 0.5;
+    const songDuration = 260;
+    const volumePosition = 70;
+    const expectedVolume = 0.7;
+
+    const player = new Player({
+      audioManager: new InMemoryAudioManager(
+        currentTime,
+        currentVolume,
+        songDuration,
+        false,
+      ),
+    });
```

Comment:
Here again… We're testing the volume feature, why should we bother with `currentTime`, `songDuration` etc… These bring just noise to the test

See what I mean?

(We'll improve it later)

Reply:
Add a TODO too.